src/apps/opening-hours-sidebar/OpeningHoursSidebarSidebar.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from "react";
+import iconWatch from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-watch-static.svg";
+import DisclosureControllable from "../../components/Disclosures/DisclosureControllable";
+import OpeningHoursSidebarSummary from "./OpeningHoursSidebarSummary";
+import OpeningHoursSidebarDetails from "./OpeningHoursSidebarDetails";
+import { LibraryType, toDayString } from "./helper";
+import { useText } from "../../core/utils/text";
+
+const OpeningHoursSidebarSidebar: FC<{ libraries: LibraryType[] }> = ({
```

Comment:
Would it make more sense to call this something like `OpeningHoursSidebarContent` instead? I think the double sidebar maybe doesn't make much sense.

Reply:
Yes, that's a good ide ☺️